### DISCUSSION TITLE
Maintained List/Catalog of Popular Apps

### DISCUSSION THREAD

**Discussion by jakariyaa** [ImranR98/Obtainium#1079] [Ideas]
I think it would be great if there were some where a maintained and moderated publicly available list of all the popular open source apps/projects. One can just easily import that list with one click and then install/update the app they wish to without needing to add sources for every single app. Or is something like this already exist that I do not know about?

**Comment by ImranR98** [comment]
This is covered in #68

**Comment by ImranR98** [comment]
See #1214.
